### DISCUSSION TITLE
Failed to run bootstrap-vcpkg.bat while calling Windows API function WinHttpReadData

### DISCUSSION THREAD

**Discussion by pentatracy5** [microsoft/vcpkg#23556] [Q&A]
I'm stuck at this bootstrap-vcpkg.bat. It says that "While calling Windows API function WinHttpReadData got error 0x00002EE2: The operation timed out". I am trying to use vcpkg in my Win10 machine. I followed every steps introduced by the Getting Started. Can some one help me please?
![image](https://user-images.githubusercontent.com/101581440/158318036-986b3635-73ce-4ffc-9a79-70a3904124a5.png)

**Comment by autoantwort** [comment]
Simply retry or do what the error message says (download manually)

**Reply by pentatracy5** [reply]
I tried but it still doesn't work...
